torchvision/prototype/features/_feature.py

```diff
@@ -8,59 +8,22 @@
 
 
 class _Feature(torch.Tensor):
-    _META_ATTRS: Set[str] = set()
-    _metadata: Dict[str, Any]
-
-    def __init_subclass__(cls) -> None:
-        """
-        For convenient copying of metadata, we store it inside a dictionary rather than multiple individual attributes.
-        By adding the metadata attributes as class annotations on subclasses of :class:`Feature`, this method adds
-        properties to have the same convenient access as regular attributes.
-
-        >>> class Foo(_Feature):
-        ...     bar: str
-        ...     baz: Optional[str]
-        >>> foo = Foo()
-        >>> foo.bar
-        >>> foo.baz
-
-        This has the additional benefit that autocomplete engines and static type checkers are aware of the metadata.
-        """
-        meta_attrs = {attr for attr in cls.__annotations__.keys() - cls.__dict__.keys() if not attr.startswith("_")}
-        for super_cls in cls.__mro__[1:]:
-            if super_cls is _Feature:
-                break
-
-            meta_attrs.update(cast(Type[_Feature], super_cls)._META_ATTRS)
-
-        cls._META_ATTRS = meta_attrs
-        for name in meta_attrs:
-            setattr(cls, name, property(cast(Callable[[F], Any], lambda self, name=name: self._metadata[name])))
-
     def __new__(
         cls: Type[F],
         data: Any,
         *,
         dtype: Optional[torch.dtype] = None,
-        device: Optional[Union[torch.device, str]] = None,
+        device: Optional[Union[torch.device, str, int]] = None,
+        requires_grad: bool = False,
     ) -> F:
-        if isinstance(device, str):
-            device = torch.device(device)
-        feature = cast(
+        return cast(
             F,
             torch.Tensor._make_subclass(
                 cast(_TensorBase, cls),
-                cls._to_tensor(data, dtype=dtype, device=device),
-                # requires_grad
-                False,
+                torch.as_tensor(data, dtype=dtype, device=device),  # type: ignore[arg-type]
```

Comment:
As discussed offline, the annotations of `as_tensor` are wrong. `str` and `int` are valid types for the `device`.